backend/src/extractors/auth.rs

```diff
@@ -50,7 +52,12 @@ async fn validate_and_fetch_uid(token: &str) -> Result<String, Error> {
     let jwk = jwks.find(&kid).expect("Specified key not found in set");
     match validate(token, jwk, validations) {
         Ok(jwt) => match jwt.claims.get("sub") {
-            Some(value) => Ok(value.to_string()),
+            Some(value) => {
+                let token =  value.as_str()
```

Comment:
You either didn't `cargo fmt` this, or we have a different config :slightly_smiling_face: